onnxruntime/hosting/server/beast_http.h

```diff
@@ -1,6 +1,9 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#ifndef BEAST_HTTP_H
```

Comment:
make it more specific? Something like `HOSTING_HTTP_BEAST_HTTP_H`?

Reply:
Done